schemas/com.snowplowanalytics.snowplow/recoveries/jsonschema/2-0-0

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an array of recovery scenarios",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "recoveries",
+    "format": "jsonschema",
+    "version": "2-0-0"
+  },
+  "type": "object",
+  "minProperties": 1,
+  "patternProperties": {
+    "^iglu:": {
+      "$ref": "#/definitions/flow"
+    }
+  },
+  "definitions": {
+    "flow": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "pattern": "^(([a-zA-Z0-9]+)(-?))+$",
+            "minLength": 1,
+            "description": "A slug name for recovery flow"
+          },
+          "conditions": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/op"
+            },
+            "description": "Conditions required to apply steps for specific Bad Row type"
+          },
+          "steps": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/op"
+            },
+            "description": "Steps to be applied for specific Bad Row type"
+          }
+        },
+        "required": [
+          "name",
+          "conditions",
+          "steps"
+        ],
+        "additionalProperties": {
+          "type": "array"
+        }
+      }
+    },
+    "op": {
+      "type": "object",
+      "properties": {
+        "op": {
+          "type": "string",
+          "enum": [
+            "Cast",
+            "Remove",
+            "Replace",
+            "Test"
+          ],
+          "description": ""
+        },
+        "path": {
+          "type": "string",
+          "pattern": "^(([a-zA-Z0-9]+)(\\.?))+$",
+          "description": ""
+        },
+        "match": {
+          "type": "string",
+          "description": ""
+        },
+        "value": {
+            "oneOf": [
+              { "type": "string" },
+              { "type": "object",
+                "description": "Compare against static value",
+                "properties": {
+                    "value": { "oneOf": [
```

Comment:
Could you please pass it through formatter https://jsonformatter.org/json-pretty-print